PolyEngine/Engine/Src/OpenGLRenderingContext.cpp

```diff
@@ -22,30 +28,30 @@ bool OpenGLRenderingContext::Init(const IRenderingContextParams * context)
 {
 	const OpenGLRenderingContextParams* openGLContext = static_cast<const OpenGLRenderingContextParams*>(context);
 	hWnd = openGLContext->HWnd; // Set the HWND for our window  
-	hDC = GetDC(hWnd); // Get the device context for our window  
+	hDC = GetDC(hWnd); // Get the device context for our window  ScreenDim.Width = openGLContext->Rect.right - openGLContext->Rect.left;
```

Comment:
Oh, so that's where the line went! Had to manually re-add it (and missed that it was still there, just in the comment). Will fix.